forch/faucetizer.py

```diff
@@ -128,6 +130,10 @@ def _is_access_port(self, port_cfg):
             property for property in non_access_port_properties if property in port_cfg]
         return len(port_properties) == 0
 
+    def _update_vlan_state(self, switch, port, state):
+        self._vlan_states.setdefault(switch, {})[port] = state
+
+    # pylint: disable=too-many-branches
```

Comment:
can you please clean up the function?  Starting to get way too big.

Reply:
Done.. Doing the initialization in another function now. PTAL..